internal/driver/driver.go

```diff
@@ -12,7 +12,7 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
-	"net/http"
+	"github.com/edgexfoundry/go-mod-bootstrap/v3/bootstrap/secret"
```

Comment:
Why using this module directly. SecrectProvider should abstract it out.

Reply:
this is because of the new `secret.WildcardName` constant I added. Do you recommend that instead we create an additional function, say `RegisterAnySecretUpdatedCallback()` that abstracts the usage out?